qa/rustfmt.txt

```diff
@@ -0,0 +1,57 @@
+rust/derive/src/applayerframetype.rs
```

Comment:
Maybe build this list within the context of the `rust/` directory, as in where its valid to run `cargo fmt`.